### PR TITLE
Load the product configuration files

### PR DESCRIPTION
**Pull request by poncovka**
Load the product configuration files to have the right defaults. Use
the product name from the os-release files of the system environment.
Simplify the product-related code.

### CODE REVIEW

File: initial_setup/product.py [hunk 1]
```diff
@@ -1,90 +1,50 @@
 """Module providing information about the installed product."""
 import logging
 
-from pyanaconda.localization import find_best_locale_match
-from pyanaconda.core.constants import DEFAULT_LANG
-import os
-import glob
-
-RELEASE_STRING_FILE = "/etc/os-release"
-LICENSE_FILE_GLOB = "/usr/share/redhat-release*/EULA*"
```
Comment (poncovka) [inline initial_setup/product.py:10]: I could find only `/usr/share/redhat-release/EULA` on the installed system.

**Comment by poncovka** [comment]
Tested with kickstart tests and a fake EULA.

**Review by jkonecny12 — COMMENTED** [review]
Looks good to me AFAICT.

**Comment by poncovka** [comment]
Rebased.

File: initial_setup/product.py (hunk 1 above)
Comment (M4rtinK) [reply initial_setup/product.py:10]: Yep, as far as I can tell this is the only file used these days, so should be correct. :)

**Review by M4rtinK — APPROVED** [review]
Looks good to me, thanks a lot - a really nice cleanup as well! :)

**Pull request merged**